src/python/ensembl/utils/rloader.py

```diff
@@ -0,0 +1,71 @@
+# See the NOTICE file distributed with this work for additional information
+#   regarding copyright ownership.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Allow to seamlessly load / read the content of a remote file as if it was located locally."""
+
+__all__ = ['RemoteFileLoader']
+
+import configparser
+import json
+import logging
+from io import StringIO
+from typing import Optional, Any
+
+import dotenv
+import requests
+import requests.exceptions as exc
+import yaml
+
+
+logger = logging.getLogger(__name__)
+
+
+class RemoteFileLoader:
```

Comment:
Missed this the first time (documentation required):
```suggestion
class RemoteFileLoader:
    """Loads remote files, allowing specific format parsing options.

    Args:
        parser: Parser to use for this object. Default, `None` (no format-specific parsing done).

    Attributes:
        available_formats (Set[str]): File formats with ad-hoc parsers available.
        parser (str): Parser selected for this object.

    """
```